package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tcomb-builder",
-  "version": "2.2.3",
+  "version": "2.3.0",
```

Comment:
This change requires a major version bump because it is not backwards compatible. Just bump it to 3.0.0 and we'll but a release after both of our PRs are merged.